cli/README.md

```diff
@@ -331,6 +341,10 @@ This is the same parameter as the one described in the [above section](#mandator
 
 This specifies the relative path to where the sourcemap file for resulting update's JS bundle should be generated. If left unspecified, sourcemaps will not be generated.
 
+### Target binary range parameter
```

Comment:
Should we mention again here what the default value is and explain that setting this parameter is valuable if you want to override that?